images/macos/provision/configuration/configure-machine.sh

```diff
@@ -17,6 +17,11 @@ if [ -d "/Library/Application Support/VMware Tools" ]; then
     sudo "/Library/Application Support/VMware Tools/vmware-resolutionSet" 1176 885
 fi
 
+# Update VoiceOver Utility to allow VoiceOver to be controlled with AppleScript by creating a special file (SIP must be disabled)
+if csrutil status | grep -Eq  "System Integrity Protection status: (disabled|unknown)"; then
```

Comment:
I was under the impression from #650 that SIP was disabled on agent images… so can this never evaluate to truthy and the step won’t ever run? Or is SIP actually/sometimes disabled during build?

Reply:
@cmorten correct, SIP is enabled and thus we enabled this setting on the image templates for now, but if someone would like to build their own VM based on our scripts this can be useful.

Reply:
Ok! Is there any guidance on building custom macOS images based on this repo, details for macOS are missing from https://github.com/actions/virtual-environments/blob/main/docs/create-image-and-azure-resources.md?

Reply:
Well, since macOS images can't be built in azure we haven't included them in this guide. We haven't created any special guide either as there were no requests for it.